build-resources.gradle

```diff
@@ -1,6 +1,7 @@
 //preferably try to main the alphabetical order
 ext.versionMap = [
-        opentelemetry_proto    : '0.10.0'
+        opentelemetry_proto    : '0.10.0',
+        es_version: '7.10.2'
```

Comment:
Can this be upgraded to 7.10.3?

Reply:
Confirmed that 7.10.3 should be 7.10 in ES branches: https://github.com/elastic/elasticsearch/blob/e65acafcd6347ae5cdd8df1d344769bd237ff88c/server/src/main/java/org/elasticsearch/Version.java#L155-L156

Will replace and check again.

Reply:
Well...actually 7.10 is not a release branch so the closest we have is still 7.10.2.